README.md

```diff
@@ -161,6 +163,14 @@ By calling `registerPageTrack`, you tell Angulartics to invoke your function on
 
 This is very similar to page tracking. Angulartics will invoke your function every time the event (`analytics-on` attribute) is fired, passing the action (`analytics-event` attribute) and an object composed of any `analytics-*` attributes you put in the element.
 
+If the analytics provider is created async, you can wrap you code with:
+
+    angulartics.waitForVendorApi("var", 1000[, "subvar"], function(window.var) {
```

Comment:
Oh, they are optional arguments. Yeah, it's not very clear you intend it to be an optional argument. Maybe just show an example without the `subvar`?